### PR TITLE
[dev-v2.9] improve documentation for pull requests

### PR DESCRIPTION
**Pull request by nicholasSUSE**
## Problem
We have noticed that no pattern is being followed in PR titles for the Charts repository. 

There are a lot of PR titles that we are reviewing that we would like to ask the developers to improve however we do not have a clear rule for it. 

Also, we noticed that the versioning rules are dubious: 

The documentation of Charts is old, without any updates for the past 1 year. 

## Solution
Fix the versioning in the [README.md file](https://github.com/rancher/charts/blob/dev-v2.9/README.md#versioning-charts)
Add the proposed rules for PR titles described in `Long Term Proposal Charts`.

### CODE REVIEW

**Review by diogoasouza — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -61,41 +71,48 @@ The X.Y.P versioning scheme roughly corresponds to the following rules (with exc
 - **Minor Version**: represents a release line of a given chart within a Rancher minor version.
 - **Patch Version**: represents a patch to a given release line of a chart within a Rancher minor version.
 
-As a rule of thumb, you will only ever touch this version to **increment the patch version once per Rancher patch release**. Once it has been incremented, it should not be incremented again until the next Rancher patch release, even if the chart points to an upstream that has been modified.
+As a rule of thumb, you will only ever touch this version to **increment the patch version once per Rancher patch release**.
+
+Once it has been incremented, it should not be incremented again until the next Rancher patch release, even if the chart points to an upstream that has been modified.
```
Comment (lucasmlp) [inline README.md]: What about OOB releases? The charts may have their patch version bumped without a new Rancher release.

File: README.md [hunk 2]
```diff
@@ -64,9 +64,9 @@ The `1.0.#` versioning scheme roughly corresponds to the following rules (with e
 - **Major Version**: represents the Rancher minor version these charts are being released to.
   - Anything less than `100`: Rancher 2.5
   - `100`: Rancher 2.6
-  - `101`: Rancher 2.7
-  - `102`: Rancher 2.7
+  - `101`and `102`: Rancher 2.7
```
Comment (lucasmlp) [inline README.md]: `101` and `102`: Rancher 2.7

File: README.md [hunk 3]
```diff
@@ -152,3 +152,38 @@ images:
...
+A working example:
+```
+[dev-v2.8] rancher-istio 103.2.0+up1.19.6 update
+```
+
+- `<chart>`: The full name of the charts exactly how it is written under `/charts folder`
+- `<version>`: The full version of the chart, exactly how it is written under `release.yaml`
+- `<action>`: `update`; `remove`; `add`
+
+What you should keep in mind for releasing charts:
```
Comment (lucasmlp) [inline README.md:171]: Are we going to add this responsibility to the teams now?

**Review by lucasmlp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nicholasSUSE) [reply README.md]: I agree this text is dubious and we still have not properly defined this. 
Let's remove it for now?

File: README.md (hunk 3 above)
Comment (nicholasSUSE) [reply README.md:171]: The teams must be aware of these rules at the time of the chart update so we don't have problems in the release merge final process.

File: README.md (hunk 1 above)
Comment (lucasmlp) [reply README.md]: Sure thing

File: README.md (hunk 3 above)
Comment (lucasmlp) [reply README.md:171]: Agreed.

**Review by lucasmlp — APPROVED** [review]
LGTM 🚀

**Pull request merged**